canonical/ubuntu-pro-client
# Make fix CLI command rely on the fix plan API endpoint

## Why is this needed?
Since we have created the fix plan API endpoint, we can now rely on it for the CLI command as well. This PR is making the necessary changes on the API endpoint itself and while also modifying the CLI command to use the API directly


## Test Steps
Run the fix integration tests and check that the output has not changed after the CLI update

## Checklist


## Does this PR require extra reviews?
